dynatrace-operator/questions.yml

```diff
@@ -164,3 +245,79 @@ questions:
         default: ""
         type: string
         show_if: "showKubernetesMonitoringCustomPropertiesSource=ValueFrom"
+
+  #################### Routing (OPTIONAL) ####################
+
+  - variable: routing.enabled
+    label: "Enable message routing"
+    description: "Enables routing for the OneAgent pods"
+    default: true
+    type: boolean
+    group: "Routing"
+    show_subquestion_if: true
+    subquestions:
+      - variable: routing.replicas
+        label: "Amount of replicas of ActiveGate pods"
+        description: "Sets the amount of replicas of ActiveGate pods are made"
+        default: 1
+        type: int
+      - variable: routing.serviceAccountName
+        label: "Name of the service-account for ActiveGate-pods"
+        description: "Optional: The name of the ServiceAccount to assign to the ActiveGate Pods."
+        default: "dynatrace-routing"
+        type: string
+      - variable: routing.tolerations
+        label: "Custom tolerations for the ActiveGate pods"
+        description: "Defines custom tolerations for the ActiveGate pods - Please edit as Yaml for the best experience - see https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/"
+        default: ""
+        type: multiline
+      - variable: routing.nodeSelector
+        label: "Node selector to control the selection of nodes"
+        description: "Defines a NodeSelector to customize to which nodes the ActiveGate will be rolled out on - Please edit as Yaml for the best experience"
+        default: ""
+        type: multiline
+      - variable: routing.resources
```

Comment:
Resources in the full-stack feature look different to here.

Reply:
Adapted to have the same structure for every resource section

Reply:
I believe you want to delete this variable then now since there is `routing_use_custom_limits_settings` below.

Reply:
Ah yes - removed